libkpod/image/copy.go

```diff
@@ -171,14 +272,20 @@ func PullImage(store storage.Store, imgName string, allTags, quiet bool, sc *typ
 	defer policyContext.Destroy()
 
 	copyOptions := common.GetCopyOptions(output, "", nil, nil, common.SigningOptions{})
-
 	for _, image := range images {
-		destRef, err := is.Transport.ParseStoreReference(store, image)
+		// Here we use a string of the source image's name instead of the string input from the user
+		// because the inputted string may have a transport pre-pended
+		//destRefName := "docker://" + srcRef.DockerReference().String()
+		destRef, err := is.Transport.ParseStoreReference(store, srcRef.DockerReference().String())
 		if err != nil {
 			return errors.Errorf("error parsing dest reference name: %v", err)
 		}
+		fmt.Printf("Trying to pull %s...", srcRef.DockerReference().String())
```

Comment:
Debug statement?  Remove.

Reply:
nope, this is doing exactly what you want.

Reply:
deferred for later